src/plugins/intel_gpu/tests/unit/transformations/decompose_reduce_scalar_output_test.cpp

```diff
@@ -0,0 +1,144 @@
+// Copyright (C) 2018-2024 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <gtest/gtest.h>
+
+#include <memory>
+#include <openvino/core/model.hpp>
+#include <openvino/opsets/opset9.hpp>
+#include <openvino/pass/manager.hpp>
+#include <plugin/transformations/decompose_reduce_scalar_output.hpp>
+#include <string>
+
+#include "common_test_utils/ov_test_utils.hpp"
+#include "intel_gpu/primitives/reduce.hpp"
+
+using namespace testing;
+using namespace ov::intel_gpu;
+using namespace cldnn;
+using ReduceType = cldnn::reduce_mode;
+
+#define create_reduce(arg, reduction, keep_dims, reduce_type)                         \
+    if (reduce_type == reduce_mode::sum)                                              \
```

Comment:
Please don't use cldnn things in transformation unit tests. You can pass op type to this macro instead of enum value:
```
#define create_reduce(arg, reduction, keep_dims, ReduceType) \
    reduce = std::make_shared<ReduceType>(arg, reduction, keep_dims);
```

Also, I'd suggest replacing macro with template

Reply:
done